utils/dbz_event_to_fhir_config.json

```diff
@@ -84,6 +196,8 @@
     "patient": {
       "enabled": "true",
       "title": "Patient",
+      "parentTable":"person",
+      "joinClause":"SELECT uuid FROM person,patient WHERE person_id = patient_id",
```

Comment:
i think for uniformity ,and making it easy to write a generic parametrized sql query , simply use the the two matching colums ie  `person_id = patient_id`